crates/config/src/prover.rs

```diff
@@ -32,6 +33,15 @@ impl ProverRunnerConfig {
             ProverType::Stwo => panic!("Stwo prover is not supported yet"),
         };
 
-        Ok(ProverRunnerConfig { prover_type, url })
+        let build_images = cliclack::select("Build and use local images?")
```

Comment:
you can just use a PromptConfirm for this
https://github.com/madara-alliance/madara-cli/blob/92a12f29957ba657439b25c119ce0775c5eb4980/crates/config/src/bootstrapper.rs#L13